guides/Web_Frameworks/AngularJS_validations.md

```diff
@@ -0,0 +1,63 @@
+# AngularJS
+
+Angular is a Javascript framework that is used to build web applications.It is primarily used for client side applications. It is Developed & Maintained by Google.
+It is mainly used for Single Page Application SPAs, where the whole page doesn't reload again and again to generate the view.
+
+### Why AngularJS?
```

Comment:
Compared to what? There is a difference between an intro guide and a book chapter 😄 
The project wants an intro guide.
Secondly, we have an article on Angular. Please check it out